scripts/arm_data_converter.py

```diff
@@ -1,8 +1,6 @@
-# -*- coding:utf-8 -*-
-import json
-import re
-import csv
+import os, json, csv, re, inspect
```

Comment:
Do you care coding style? if yes ...

In Python coding style guide (known as PEP8 or any others)
`Imports should usually be on separate lines:`
One of reasons, when developers read the DIFF, separate lines import is better.

This may be mixed another script languages cultures.
It's ok little thing since this project does not have Contributing guide lines.

Reply:
I work mostly in Webstorm for motocal I didn't run the python scripts with auto format and auto optimize with Pycharm, but with Webstorm and that doesn't do formating correctly for whatever reason.

Reply:
Any Python IDE/editors can warn unused module imports. (or maybe extra option for Linter)
I saw now 7e7c2b8 some unused imports are removed. 👍